Sources/App/Models/Heartbeat/Heartbeat.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Heartbeat.swift
+//  App
+//
+//  Created by di on 29.11.2017.
+//
+
+import FluentProvider
+
+// sourcery: AutoModelGeneratable
+// sourcery: toJSON, Preparation
+final class Heartbeat: Model {
+  let storage = Storage()
+  
+  var beat: Int
+  
+  init(beat: Int) {
+    self.beat = beat
+  }
+  
+
+// sourcery:inline:auto:Heartbeat.AutoModelGeneratable
```

Comment:
Small note: move this annotation(and `// sourcery: end` too) 2 spaces to the right. so it will be on one level with `init(row: Row)` function.

Reply:
fixed